implementations/micrometer-registry-elastic/src/test/java/io/micrometer/elastic/ElasticMeterRegistryTest.java

```diff
@@ -257,6 +257,12 @@ void getVersionWhenVersionIs5() {
         assertThat(ElasticMeterRegistry.getMajorVersion(responseBody)).isEqualTo(5);
     }
 
+    @Issue("#1505")
+    @Test
+    void getVersionWhenVersionIs5_3_0() {
```

Comment:
It'd be good to rename to something the test method does like`getVersionWhenVersionIs5AndNotPrettyPrinted`.